CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ if("${ZIG_VERSION}" STREQUAL "")
     endif()
 endif()
 message("Configuring zig version ${ZIG_VERSION}")
+file(WRITE ${CMAKE_BINARY_DIR}/zig_version.txt ${ZIG_VERSION})
```

Comment:
I'm not ok with this, let's not rely on cmake to write the zig version. surely it can be solved a better way.